src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -192,40 +216,44 @@ class HoldRequest extends React.Component {
     return '';
   }
 
-  /**
-   * renderEDD()
-   * Renders the radio input fields of EDD.
-   *
-   * @return {HTML Element}
-   */
-  renderEDD() {
-    return (
-      <label
-        className="electronic-delivery"
-        id="radiobutton-group1_electronic-delivery"
-        htmlFor="available-electronic-delivery"
-      >
-        <input
-          aria-labelledby="radiobutton-group1 radiobutton-group1_electronic-delivery"
-          type="radio"
-          name="delivery-location"
-          id="available-electronic-delivery"
-          value="edd"
-          checked={this.state.checkedLocNum === -1}
-          onChange={e => this.onRadioSelect(e, -1)}
-        />
-        Have up to 50 pages scanned and sent to you via electronic mail.
-      </label>
-    );
+  // redirects if patron is ineligible to place holds
+
+  conditionallyRedirect() {
+    return this.checkEligibility(this.state.patron.id).then((eligibility) => {
+      clearTimeout(this.timeoutId);
+      if (!eligibility.eligibility) {
+        const bib = (this.props.bib && !_isEmpty(this.props.bib)) ?
+          this.props.bib : null;
+        const bibId = (bib && bib['@id'] && typeof bib['@id'] === 'string') ?
+          bib['@id'].substring(4) : '';
+        const itemId = (this.props.params && this.props.params.itemId) ? this.props.params.itemId : '';
+        const path = `${appConfig.baseUrl}/hold/confirmation/${bibId}-${itemId}`;
+        return this.redirectWithErrors(path, 'eligibility', JSON.stringify(eligibility));
+      }
+      this.updateIsLoadingState(false);
+    });
+  }
+  // checks whether a patron is eligible to place a hold
+  checkEligibility() {
+    return new Promise((resolve) => {
+      axios.get(`${appConfig.baseUrl}/api/patronEligibility`)
+        .then((response) => { resolve(response.data); });
+    });
   }
 
+  redirectWithErrors(path, status, message) {
+    this.context.router.replace(
+      `${path}?errorStatus=${status}` +
+      `&errorMessage=${message}`,
+    );
+  }
   /**
-   * renderDeliveryLocation(deliveryLocations = [])
-   * Renders the radio input fields of delivery locations except EDD.
-   *
-   * @param {Array} deliveryLocations
-   * @return {HTML Element}
-   */
+     * renderDeliveryLocation(deliveryLocations = [])
```

Comment:
This was aligned correctly before.

Reply:
Hmmm that's odd my linter is happy with it

Reply:
The linter won't pick that up, but that's the convention used for esdoc.